easybuild/easyconfigs/x/XZ/XZ-5.2.2-GCCcore-4.9.3.eb

```diff
@@ -0,0 +1,27 @@
+easyblock = 'ConfigureMake'
+
+name = 'XZ'
+version = '5.2.2'
+
+homepage = 'http://tukaani.org/xz/'
+description = "xz: XZ utilities"
+
+toolchain = {'name': 'GCCcore', 'version': '4.9.3'}
+
+sources = [SOURCELOWER_TAR_BZ2]
+source_urls = ['http://tukaani.org/xz/']
+
+builddependencies = [
+    ('Autotools', '20150215'),
+    ('gettext', '0.19.8', '', True),
```

Comment:
@akesandgren please include `binutils` as a build dep too here, otherwise this fails with the following (from `config.log`):

```
collect2: fatal error: cannot find 'ld'  
``` 

```python
builddependencies = [
     ('binutils', '2.25'),
 ]
```